docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -2,30 +2,53 @@
 
 It may be desired to setup a dev cluster having one or more worker nodes in order to play with comprehensive features of Arktos. The simple and easy-to-use arktos-up.sh feels short in this case; this doc describes the minimum effort to run a cluster having 1 master node and extra worker nodes joining after the master is up.
 
-This doc, as at the moment written, does not mandate which cni plugin to be used; it is up to the user. We have verified that Flannel works well with multi-tenancy Arktos cluster; the instructions laid out here are based on our experience with Flannel at GCP env.
+This doc, as at the moment written, does not mandate which cni plugin to be used; it is up to the user. We have verified that Flannel works well with multi-tenancy Arktos cluster; the instructions laid out here are based on our experience with Flannel at GCP env or AWS env with Ubuntu 18.04 x86 image.
 
-Assuming you have got the Arktos repo downloaded to your local disk and the current folder is at the root of the repo,
+Assuming you have got the Arktos repo downloaded to your local disk and the current folder (i.e ~/go/src/arktos/) is at the root of the repo, and 
 
-0. Make sure the following directories are empty. If not, clean them up.
+- Setup master node and workwer nodes based on [set up developer environment](setup-dev-env.md) to install needed packages (docker, make, gcc, jq and golang).
+- SSH/SCP should work from worker nodes to master node in order to copy the worker secret files from the master node (in AWS, use private key file of keypair to access to master/worker nodes).
+- Needed ports are opened between master node and worker nodes
+  * Allow access to kube-api port 6443 on master node from work nodes (in AWS, add this rule into inbound rules of security group for master node)
+  * Allow access to kubelet port 10251 and kube-proxy port 10255 on worker nodes from master node(in AWS, add this rule into inbound rules of security group for worker nodes)
+- On master node, the permisson of others for file /var/run/docker.sock should be readable and writable.
+
+```bash
+srw-rw-rw- 1 root docker 0 Aug  3 22:15 /var/run/docker.sock
+```
```

Comment:
please indicate this is the output of "ls" command

Reply:
Add more explanations. Thanks!